import i18next from 'i18next';

i18next.addResources('es', 'ERRORBOUNDARY', {
  TITLE: '¡Oh no! Ha ocurrido un error',
  TRY_AGAIN: 'Intenta nuevamente',
  BACK_TO_HOME: 'Volver al inicio',
  ERROR_ALERT: 'Ocurrió un error inesperado',
  TYPE_ERROR: `Error: {{fatal}}`,
  ERROR_MESSAGE:
    '\n\n¡Hemos informado de esto a nuestro equipo! Cierre la aplicación e inicie nuevamente'
});
